README.md

```diff
@@ -114,6 +114,16 @@ stack install
 
 ```
 
+#### nix (any linux distribution)
```

Comment:
Should work on macOS too, right?

Reply:
@iclanzan To be honest I am not sure. Would need someone to test it.